.github/workflows/ci_test-full.yml

```
@@ -33,8 +33,7 @@ jobs:
 
     - name: Update Pip
       run: |
-        # todo: unfreeze PIP after resolving minimal dependencies
-        pip install --quiet "pip==20.1" --upgrade --user  # needed for get pip cacher folder
+        pip install --quiet "pip>=20.3" --upgrade --user  # needed for get pip cacher folder
```

@Borda Sorry for the delay.

How about always using the latest version rather than pinning the min version? `pip` only needs >=20.1 for caching ([blog post](https://blog.python.org/2020/04/pip-20-1-released.html)), and using the latest version won't affect users at all, right?
```suggestion
        pip install --quiet pip --upgrade --user  # needed for get pip cacher folder
```